changes/6477.bugfix

```diff
@@ -0,0 +1 @@
+Type of uploads for group and user image can be restricted via `ckan.upload.user.types` and `ckan.upload.group.types`
```

Comment:
```suggestion
Type of uploads for group and user imagec can be restricted via the `ckan.upload.{object_type}.types` and `ckan.upload.{object_type}.mimetypes` config options (eg `ckan.upload.group.types`, `ckan.upload.user.mimetypes`)
```

If you can link to the `doc/configuration.rst` docs with `:ref:` that would be ideal